metagraph/src/cli/config/config.hpp

```diff
@@ -182,6 +182,7 @@ class Config {
         RowDiff,
         RowDiffBRWT,
         RowFlat,
+        RowSparse,
```

Comment:
Maybe RowEnc? But RowSparse is fine too.

Reply:
I don't mind either way. I don't like RowSparse very much, I don't dislike RowEnc more :)